salt/journal/config/srv-journal-docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+    redis:
+        image: redis:3.2.10-alpine
+    fpm:
+        image: "elifesciences/journal:${IMAGE_TAG}"
+        volumes:
+            - ./parameters.yml:/srv/journal/app/config/parameters.yml
+            - ./var/logs:/srv/journal/var/logs
+        ports:
+            - "9000:9000"
+        env_file:
+            - ./containers.env
+        restart: always
+        volumes:
+            - ./parameters.yml:/srv/journal/app/config/parameters.yml
+        depends_on:
+            - redis
+    assets:
+        image: "elifesciences/journal_web:${IMAGE_TAG}"
```

Comment:
Can we avoid having two published images? Refs https://github.com/elifesciences/journal/pull/1032.

Reply:
I couldn't find the resolution of the original issues, so with this new information, maybe